src/pages/index.tsx

```diff
@@ -36,15 +37,27 @@ const IndexPage: React.FC<PageProps> = ({ location }) => {
       </Hero>
       <div className="p-5">
         <SplitContent
-          imageUrl="https://upload.wikimedia.org/wikipedia/commons/thumb/e/e7/RS1_Radschnellweg_Ruhr_in_M%C3%BClheim_2.jpg/800px-RS1_Radschnellweg_Ruhr_in_M%C3%BClheim_2.jpg?uselang=de"
+          image={
+            <StaticImage
+              className="h-full w-full object-cover"
+              src="https://upload.wikimedia.org/wikipedia/commons/thumb/e/e7/RS1_Radschnellweg_Ruhr_in_M%C3%BClheim_2.jpg/800px-RS1_Radschnellweg_Ruhr_in_M%C3%BClheim_2.jpg?uselang=de"
+              alt="Support team"
+            />
+          }
```

Comment:
I think in this and the next case we should add a wrapper HTML Element in the component instead of pass the `className`s.

Reply:
"add a wrapper HTML Element" – you mean a wrapper component? Because the tags on a wrapper element would mean something different than on the img-tag-